### DISCUSSION TITLE
Use values from original request body to take authorization decision with "authorizer remote". Getting Oathkeeper error "Read on closed Body"

### DISCUSSION THREAD

**Discussion by denysandriyanov** [ory/oathkeeper#1135] [Q&A] [answered]
Hello all

Kindly asking for your help as cant figure out myself.

I want to use a remote authorizer where i can take authorization decisions based on values in the original request body. I implemented authorizer that recieves POST request with the body and returns either 200 or 403. But when I return 200 I am getting "502 Bad Gateway".

In Oauthkeeper logs i see below:
time=2023-09-09T10:35:35Z level=error msg=http: gateway error audience=application error=map[message:readfrom tcp 10.81.28.36:35082->10.0.214.228:80: http: invalid Read on closed Body] service_name=ORY Oathkeeper service_version=v0.40.6
[11:55](https://ory-community.slack.com/archives/C01340V8KSM/p1694256904100229)

Please help me figure out what I am doing wrong.
Thank you!

**Comment by denysandriyanov** [comment, marked as answer]
Logged a bug https://github.com/ory/oathkeeper/issues/1136

**Reply by vinckr** [reply]
Hey @denysandriyanov 
Thanks! Lets continue in the isse ✅
